pkg/cmd/backuprestore/backupserver.go

```diff
@@ -52,6 +59,24 @@ func (b *backupServer) AddFlags(fs *pflag.FlagSet) {
 }
 
 func (b *backupServer) Validate() error {
+	if !b.enabled {
+		klog.Infof("backup-server is disabled")
+		return nil
+	}
+
+	if len(b.schedule) == 0 {
+		err := errors.New("missing required flag: --schedule")
```

Comment:
if you're using cobra now, you can mark it required.

I would also assume validate to try to parse the cron and return a more explicit error here

Reply:
done